interpreter/runtime/table.mli

```diff
@@ -11,11 +11,12 @@ exception Type
 exception Bounds
 exception SizeOverflow
 exception SizeLimit
+exception OutOfMemory
 
-val alloc : table_type -> table
+val alloc : table_type -> ref_ -> table
 val type_of : table -> table_type
 val size : table -> size
-val grow : table -> size -> unit (* raises SizeOverflow, SizeLimit *)
+val grow : table -> size -> ref_ -> unit (* raises SizeOverflow, SizeLimit *)
```

Comment:
Nit: add OutOfMemory to the comment here?

Reply:
Done.